packages/contentful/migrations/crn/manuscripts/20250519081547-add-apc-fields.js

```diff
@@ -0,0 +1,90 @@
+module.exports.description =
+  'Add APC fields and rename apcAmount to apcAmountPaid on Manuscript model';
+
+module.exports.up = (migration) => {
+  const manuscript = migration.editContentType('manuscripts');
+
+  manuscript
+    .createField('apcCoverageRequestStatus')
+    .name('APC Coverage Request Status')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([{ in: ['notPaid', 'paid', 'declined'] }])
+    .disabled(false)
+    .omitted(false);
+
+  manuscript
+    .createField('declinedReason')
+    .name('Declined Reason')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([{ size: { max: 100 } }])
+    .disabled(false)
+    .omitted(false);
+
+  manuscript
+    .createField('apcAmountRequested')
+    .name('APC Amount Requested')
+    .type('Number')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  manuscript
+    .createField('apcAmountPaid')
+    .name('APC Amount Paid')
+    .type('Number')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  // Note: No data migration is required. The field apcAmount was unused.
+  manuscript.deleteField('apcAmount');
+  manuscripts.deleteField('apcPaid');
+
+  manuscripts.moveField('declinedReason').afterField('apcRequested');
+  manuscripts.moveField('apcAmountPaid').afterField('apcRequested');
+  manuscripts.moveField('apcCoverageRequestStatus').afterField('apcRequested');
+  manuscripts.moveField('apcAmountRequested').afterField('apcRequested');
```

Comment:
Does it work to leave all `afterField('afterField')`? I've never tried 😛 

Anyway, I think it would be nice to leave the order more or less as it appears in the modal in the frontend (https://www.figma.com/design/zY4fHBAsXCiyW6gGrnS4Yy/CRN---Design---Network?node-id=34790-352904&t=51uKJvYgGEpm96IJ-4)

```suggestion
  manuscripts.moveField('apcAmountRequested').afterField('apcRequested');
  manuscripts.moveField('apcCoverageRequestStatus').afterField('apcAmountRequested');
  manuscripts.moveField('apcAmountPaid').afterField('apcCoverageRequestStatus');
  manuscripts.moveField('declinedReason').afterField('apcAmountPaid');
```